crates/subspace-core-primitives/src/lib.rs

```diff
@@ -489,7 +489,7 @@ impl SegmentHeader {
 }
 
 /// Sector index in consensus
```

Comment:
Do we have a significant improvement here? If we change the consensus again we could forget to increase SectorIndex size and have problems.

Reply:
I do not think we'll forget, this type alias (will have to change to new type with additional constants and methods in the future) is used in multiple places, it should be quite clear if it no longer works. But I also don't think we'll need to change it. Also we have sector size as consensus parameter and it is likely that we'll increase sector size over time, so farmer will be able to plot bigger and bigger plots in the future with the same number of sectors.